integration_tests/fixtures/applicationData.json

```diff
@@ -114,19 +114,22 @@
     }
   },
   "risk-to-self": {
-    "guidance": {},
     "vulnerability": {
       "vulnerabilityDetail": "example answer vulnerability",
-      "dateOfOasysImport": "2023-08-31"
+      "dateOfOasysImport": "2023-08-31",
+      "confirmation": "confirmed"
```

Comment:
I think adding `confirmation` has broken these three integration tests - it looks like those tests just need to delete this field when the `application` is being set up.

Reply:
I've actually removed the error handling integration tests because I think we decided to rely on our unit tests for these cases?